inc/classes/class-assets.php

```diff
@@ -45,9 +56,9 @@ protected function _setup_hooks() {
 	 */
 	public function register_scripts() {
 
-		wp_register_script( 'blank-theme-main', BLANK_THEME_BUILD_URI . '/js/main.js', [ 'jquery' ], false, true );
-		wp_register_script( 'blank-theme-home', BLANK_THEME_BUILD_URI . '/js/home.js', [ 'jquery' ], false, true );
-		wp_register_script( 'blank-theme-single', BLANK_THEME_BUILD_URI . '/js/single.js', [ 'jquery' ], false, true );
+		wp_register_script( 'blank-theme-main', $this->get_asset_file_uri( 'main.js' ), [ 'jquery' ], filemtime( $this->get_asset_file_path( 'main.js' ) ), true );
```

Comment:
I am glad you created a function get_asset_file_uri() to save rework.
I created something like this for one our client's project

Reply:
yep the aim is to reduce the rework that we do after cloning the blank-theme, :)

suggestions are welcome, we can create issues and work on whenever get time.